utils.js

```diff
@@ -183,25 +183,60 @@ CTLEventUtils.readURLParams = function(eventsList, queryString, index) {
             case 'q':
                 filteredEvents = CTLEventUtils.searchEvents(
                     filteredEvents, index, splitParam[1]);
+                CTLEventUtils.populateQ(splitParam[1]);
```

Comment:
`readURLParams` returns an array of event objects, so a `populateInputs()` wouldn't have the info needed to fill in the fields.  I can imagine a stand-alone `populateInputs()` that would work just like `readURLParams()` minus the filtering.